tests/Folder/FolderManagerTest.php

```diff
@@ -399,4 +447,20 @@ public function testGetFolderPermissionsForUserMerge(): void {
 		$permissions = $manager->getFolderPermissionsForUser($this->getUser(['g1', 'g2', 'g3']), 2);
 		$this->assertEquals(0, $permissions);
 	}
+
+	public function testQuotaDefaultValue() {
+		$rootStorageId = Server::get(IRootFolder::class)->getMountPoint()->getNumericStorageId();
+		$folderId1 = $this->manager->createFolder('foo');
+
+		$exponent = 3;
+		$this->config->expects($this->any())
+			->method('getSystemValueInt')
+			->with('groupfolders.quota.default', FileInfo::SPACE_UNLIMITED)
+			->willReturnCallback(function () use (&$exponent) {
+				return 1024 ** ($exponent++);
+			});
+
+		$this->assertEquals(1024 ** 3, $this->manager->getFolder($folderId1, $rootStorageId)['quota']);
+		$this->assertEquals(1024 ** 4, $this->manager->getFolder($folderId1, $rootStorageId)['quota']);
+	}
```

Comment:
For some reason adding this test makes the trash tests fail. Keeping everything the same and just removing this test makes them work again.
I tried to debug this for a while, but couldn't make any sense of it.
The difference for the trash tests is that the permissions get calculated differently because a rule is not added which causes the tests to fail as they expect the normal user to no longer see the file without read permission.